pages/common/forever.md

```diff
@@ -0,0 +1,15 @@
+# forever - nodejs tool
+
+> Server-side JavaScript application that makes sure node applications run indefinitely (restarts after crash).
```

Comment:
I havent really used `forever` before. Does it only restart after crashes or also just normal exiting (i.e. exit code 0)? If so maybe its better to say `(restarts after exit).`

Reply:
It also restarts after a clean exit (0). I guess you're right, it could say "exit".